content/400-reference/300-database-reference/02-connection-urls.mdx

```diff
@@ -21,6 +21,20 @@ Make sure you have this information at hand when getting started with Prisma. If
 
 </TopBlock>
 
+## Format
+
+The format of the connection URL depends on the _database connector_ you're using. Prisma generally supports the standard formats for each database. You can find out more about the connection URL of your database on the dedicated docs page:
+
+- [PostgreSQL](/concepts/database-connectors/postgresql)
+- [MySQL](/concepts/database-connectors/mysql)
+- [SQLite](/concepts/database-connectors/sqlite)
+- [MongoDB](/concepts/database-connectors/mongodb)
+- [Microsoft SQL Server](/concepts/database-connectors/sql-server)
+
+### Special characters
+
+For MySQL and PostgreSQL, you must percentage-encode special characters in any part of your connection URL - including passwords. For example, `p@$$w0rd` becomes `p%40%24%24w0rd`.
```

Comment:
Might be good to link a reference here (e.g. https://developer.mozilla.org/en-US/docs/Glossary/percent-encoding) so people can look up the codes?

Otherwise change looks good.

Reply:
That's a great idea:
```suggestion
For MySQL and PostgreSQL, you must [percentage-encode special characters](https://developer.mozilla.org/en-US/docs/Glossary/percent-encoding) in any part of your connection URL - including passwords. For example, `p@$$w0rd` becomes `p%40%24%24w0rd`.
```
Feel free to accept this edit suggestions, make another one or just commit to my branch yourself @keerlu 👍